deploy/roles/default_role.yaml

```diff
@@ -45,12 +45,6 @@ rules:
     resources:
       - configmaps
     resourceNames: ["assisted-service-leader-election-helper"]
-  - verbs:
```

Comment:
What does it affect?

Reply:
nothing

Reply:
that's why i remove it. Looks like it was bug to add it

Reply:
Did you check it on PSI? (that it's not breaking anything)

Reply:
sure, i did

Reply:
Great. 
What about app-sre? do we need to change something there? or ask someone to remove this permission?

Reply:
I am on the app-sre side @filanov

Reply:
https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/9126
Waiting for this PR to be merged before removing the `/hold` from the MR.